README.md

```diff
@@ -1,75 +1,132 @@
+## Interface Overview 📖
 
-```
-npm run lint:ts-fix
-```
+The DLT Interface consists of several key events and functions that facilitate the handling and management of tokens.
 
-This command runs the ESLint linter on all TypeScript files in the project and automatically fix any issues it finds.
+Key events include:
 
-### Lint Solidity
+- `Transfer`: Emitted when a token is transferred.
+- `TransferBatch`: Emitted for batch transfers.
+- `Approval`: Emitted when an owner approves a spender to manage a token.
+- `ApprovalForAll`: Emitted when a spender enables or disables an operator to manage all of its assets.
+- `URI`: Emitted when the URI of a mainId is changed.
 
-```
-npm run lint:sol
-```
+Key functions include:
 
-This command runs the Prettier and Solhint linters on all the Solidity files in the project. It will execute the command `npx prettier '**/*.{json,sol,md}' --check && npx solhint 'contracts/**/*.sol'`, which will check all the solidity files in the project against a set of linting rules and report any errors or warnings.
+- `setApprovalForAll`: Approve or remove an operator for the caller.
+- `safeTransferFrom`: Moves tokens using the allowance mechanism.
+- `approve`: Sets the allowance of a spender over the caller's tokens.
+- `subBalanceOf`: Returns the amount of tokens owned by an account.
+- `balanceOfBatch`: Returns the balances of multiple accounts.
+- `allowance`: Returns the remaining number of tokens that a spender can spend on behalf of an owner.
+- `isApprovedForAll`: Checks if an operator is allowed to manage all of the assets of an owner.
 
-### Lint Solidity and fix issues
+## Key Functions 📚
 
-```
-npm run lint:sol-fix
-```
+The DLT interface provides a set of functions to interact with and manage the tokens. Here's a brief overview:
 
-This command runs the Prettier and Solhint linters on all the Solidity files in the project and automatically fix any issues it finds.
+- `setApprovalForAll(operator, approved)`: Allows the approval or removal of `operator` as an operator for the caller. Operators can call `transferFrom` or `safeTransferFrom` for any token owned by the caller.
+- `safeTransferFrom(sender, recipient, mainId, subId, amount, data)`: Moves `amount` tokens from `sender` to `recipient` using the allowance mechanism. `amount` is then deducted from the caller's allowance.
+- `approve(spender, mainId, subId, amount)`: Sets `amount` as the allowance of `spender` over the caller's tokens.
+- `subBalanceOf(account, mainId, subId)`: Returns the amount of tokens owned by `account
+- `balanceOfBatch(accounts, mainIds, subIds)`: Returns the balances of multiple `accounts` for each pair of `mainIds` and `subIds`.
+- `allowance(owner, spender, mainId, subId)`: Returns the remaining number of tokens that `spender` can spend on behalf of `owner` for a specific `mainId` and `subId`.
+- `isApprovedForAll(owner, operator)`: Checks if `operator` is allowed to manage all of the assets of `owner`.
```

Comment:
Are we coloring the methods and the parameters ? or we are coloring the normal words.

Methods in blue and others in black

Reply:
![Screenshot 2023-05-16 at 7 51 07 PM](https://github.com/polytrade-finance/dual-layer-token/assets/22845388/c09703ef-d4be-40a0-8185-e812177960d5)
it's an issue on the visualizer, only the functions variables and functions names are highlited